Makefile

```diff
@@ -55,3 +56,13 @@ dist: clean
 	python setup.py sdist
 	python setup.py bdist_wheel
 	ls -l dist
+
+upgrade: export CUSTOM_COMPILE_COMMAND=make upgrade
+upgrade:
+	pip install -qr requirements/pip-tools.txt
+	pip-compile --upgrade -o requirements/pip-tools.txt requirements/pip-tools.in
+	pip-compile --upgrade -o requirements/base.txt requirements/base.in
+	pip-compile --upgrade -o requirements/test.txt requirements/test.in
+	pip-compile --upgrade -o requirements/dev.txt requirements/dev.in
+	# Let tox control the Django version for tests
+	sed '/^[dD]jango==/d' requirements/test.txt > requirements/tox.txt
```

Comment:
This isn't what we normally use `tox.txt` for; typically `tox.in` and `tox.txt` are for the dependencies to run tox itself, and `test.txt` is updated here to exclude Django.

Reply:
i will update that.